ext/bg/js/audio.js

```diff
@@ -112,6 +115,24 @@ async function audioBuildUrl(definition, mode, cache={}) {
     }
 }
 
+function normalizeAudioUrl(url, baseUrl, basePath) {
```

Comment:
Please rename to `audioUrlNormalize` to be consistent with all the other functions in the file, which have the `audio` prefix.